### ISSUE TITLE
The loss function of critic

### ISSUE THREAD

**Issue by Breezensr** [Zardinality/WGAN-tensorflow#14]
`c_loss = tf.reduce_mean(fake_logit - true_logit)` maybe wrong. It should be
 `c_loss = tf.reduce_mean(fake_logit) - tf.reduce_mean(true_logit)`

**Comment by libing125** [comment]
I think they are the same.
